clients/gateway.go

```diff
@@ -60,6 +64,62 @@ func NopBackoff(d time.Duration) time.Duration {
 	return 0
 }
 
+// NewTestGatewayClient returns a client and a function to close a test server.
+func NewTestGatewayClient(network utils.Network) (*GatewayClient, func()) {
```

Comment:
Remove stutter from `NewTestGatewayClient` and `NewGatewayClient` by renaming them to `NewTestGateway` and `NewGateway`, respectively.